README.md

```diff
@@ -0,0 +1,24 @@
+# Satellite Ansible Roles
+### Note: The roles here are not officially supported by Red Hat.
+
+Ansible Roles for interacting with the Red Hat Satellite API, combining different modules into workflows. The roles will be created using [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) (FAM).
+
+# What this project is not?
+- This is not an Ansible Collection. The roles present here are standalone roles.
+- This will not be published in Ansible Galaxy.
+- This project will not accept Ansible modules. If you are writing a new module, consider contributing it to FAM.
+- The roles here will not be shipped with FAM. If you like your role to be shipped with foreman-ansible-modules, consider contributing it to FAM using the rules listed in [Link to be updated later] (https://github.com/theforeman/foreman-ansible-modules/issues/646)
```

Comment:
It might be overkill, but I also want to push folks to FAM as much as possible, thus turning each refrence to FAM into a hyperlink to the FAM website (or at lest the first one in the bullet point) would be nice.

Reply:
Sure, I added a few more hyperlinks to FAM.